public/css/common.less

```diff
@@ -95,23 +95,23 @@ div.show-more {
     margin-bottom: .5em;
   }
 
+  // ToDo: .5em spacing (`margin-bottom`) should be applied to `.collapsible` content?
+  > .collapsible {
+    margin-bottom: 0
+  }
```

Comment:
Too generic for me. The only reason why this is needed is due to the host meta info widget. This widget already got the bottom margin, so it should be removed in the same place due to the same circumstances. (Then possibly by targeting the id again, my previous comment against the id was flawed)

Reply:
As discussed, I used the id.

Reply:
✅